azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -0,0 +1,363 @@
+# coding=utf-8
```

Comment:
if it is just OPCUA connector stuff (and no plans for other connectors) - you can flatten the structure temporarily. We can restructure code inside later

Reply:
there is plan with other connector, confirmed with Paymaun